core/utils_test.go

```diff
@@ -320,13 +324,42 @@ func TestNextVersionOptionsWithAuto(t *testing.T) {
 		{
 			Kind: "minor",
 		},
+		{
+			Kind: "skip",
+		},
 	}
 
 	ver, err := GetNextVersion(mockRead, config, "auto", nil, nil, changes, "")
 	then.Nil(t, err)
 	then.Equals(t, "v0.3.0", ver.Original())
 }
 
+func TestNextVersionOptionsNoneAutoLevelOnly(t *testing.T) {
+	config := &Config{
+		Kinds: []KindConfig{
+			{
+				Label:     "skip",
+				AutoLevel: NoneLevel,
+			},
+		},
+	}
+	latestVersion := "v0.2.3"
+	mockRead := func(dirname string) ([]os.DirEntry, error) {
+		return []os.DirEntry{
+			&then.MockDirEntry{MockName: latestVersion + ".md"},
+		}, nil
+	}
+	changes := []Change{
+		{
+			Kind: "skip",
+		},
+	}
+
+	ver, err := GetNextVersion(mockRead, config, "auto", nil, nil, changes, "")
+	then.Equals(t, ver, nil)
```

Comment:
```suggestion
	then.Nil(t, ver)
```
There is a nil shortcut in then.

Reply:
> There is a nil shortcut in then.
```
--- FAIL: TestNextVersionOptionsNoneAutoLevelOnly (0.00s)
    utils_test.go:359: expected '<nil>' to be nil
```
Not sure why, but this seems to be a case for `in some cases you may need to do 'Equals(t, value, nil)'`